packages/main/src/ComboBoxFilters.js

```diff
@@ -30,4 +30,11 @@ const Contains = (value, items) => {
 	});
 };
 
-export { StartsWithPerTerm, StartsWith, Contains };
+const None = (_, items) => items;
+
+export {
+	StartsWithPerTerm,
+	StartsWith,
+	Contains,
+	None,
```

Comment:
Is there a real use-case for this filter?

Reply:
In case the user wants to have a free text without filtering or implement filtering on the backend this would be a solution.